docs/reference/gettingstartedexample/gse-debug-guide.md

```diff
@@ -0,0 +1,232 @@
+- **dram status** - This indicator gives you insight into the state of the DRAM manager.
+   - **bytes written** - Number of bytes written to this channel's DRAM partition.
+   - **bytes read** - Number of bytes read from this channel's DRAM partition.
+   - **overflow** - The DRAM parition could not be written to because the partition was full.
+   - **partition full** - A live status showing when the parition is full. The partition can be full, it just needs to be read from before additional writes are made.
+   - **last byte flushed** - An end of acquisition status where the last valid byte of the last packet has been read from the DRAM partition.
+
+
+What kinds of scenarios can you sniff out based on the indicator behavior?
+- If **acq state** is stuck in idle, we are never detecting a valid CSI-2 packet
+- If **stop timed out** is true, we did not receive the last packet in the expected amount of time. Increase the **Stop Timeout (s)** control on the front panel to see if the packet is late, or if there is a missing packet.
+- If any of the bad packet indicators are lit, we are not receiving valid CSI-2 packets. Check your camera and connection to the serial input channel.
+- If the acquisition has ended but **bytes written** is greater than **bytes read** then there is valid data getting stuck in the DRAM partition and something downstream is not ready to receive CSI-2 packets.
+- If the **overflow** indicator is true, the DRAM manager and data path is not keeping up with the incoming packets on the serial input channel. This can be caused from the FPGA to Host FIFO backing up and pushing back on the DRAM manager. This can happen if the data is not being written to disk fast enough on the host or if data is not being pulled out of the serial channel FIFO fast enough on the host side.
+- If the **partition full** indicator is true, the partition needs to be read from. You can instrument this signal on the FPGA side to see how often the partition is getting full. It likely bounces between full and not full many times before finally overflowing. This is indicative of the host FIFO or data path not being able to keep up with incoming data on the serial input channel.
+- If the **last byte flushed** stays false when an acquisition completes, the last packet will be missing from the data set.
+
+
+### Get serial output channel status from the FPGA
+One of the best ways to diagnose Generation errors is to monitor the Serial Output Channel Status indicator from the FPGA. This indicator can be added to the Generation GSE from the Read Serial Output Channel Status VI.
+
+![Serial Output Channel Status](../../images/PXIe-148X-Serial-Output-Status.PNG)
+
+The Serial Output Channel Status is an array of status cluster indicators - one for each channel.
+The cluster has the following indicators:
+- **num buffered llp packets** - Indicates the number of llp packets that are waiting to be generated.
```

Comment:
I know LLP is lowercase on the indicator label because that is our convention, but I think it should be uppercase in the written description since it is an acronym.  This applies to all instances of LLP and CSI-2.